packages/sources/tiingo/README.md

```diff
@@ -217,7 +227,7 @@ Request:
 
 ## Crypto-lwba Endpoint
 
-Supported names for this endpoint are: `crypto-lwba`, `crypto_lwba`, `cryptolwba`, `price`.
+Supported names for this endpoint are: `crypto-lwba`, `crypto_lwba`, `cryptolwba`.
```

Comment:
I can't see why `price` is being removed here - is this just because the README was out of date?

Reply:
Probably. I don't see any `price` alias for lwba endpoint.

Reply:
It's strange. 3 days ago the readme was updated by the release script. This should have been removed there, if not before. I'll check what's the issue (not related to this PR)